README.md

```diff
@@ -20,20 +20,20 @@ Please make sure you have the following installed:
 
 Next run this command to setup OpenMRS server (just the first time):
 ````sh
-$ mvn openmrs-sdk:setup -DserverId=refapp -Dversion=2.3
+$ mvn openmrs-sdk:setup -DserverId=refapp -Ddistro=referenceapplication:2.4
 # Note: Pick default values for everything except MySQL username and password
 ````
```

Comment:
Since we need owa module I would add one more step
> mvn openmrs-sdk:deploy -DserverId=server9 -DartifactId=owa -Dversion=1.6.3 -DgroupId=org.openmrs.module